### PR TITLE
Round two of fixes for random eye-sores

### PR DESCRIPTION
**Pull request by OtterBoops**

### CODE REVIEW

**Comment by OtterBoops** [comment]
I can't find anything else that needs fixing right now.

File: dashboard/src/components/dashboard/Dashboard.vue [hunk 1]
```diff
@@ -60,6 +60,6 @@ export default {
 
 <style scoped lang="scss">
 .Dashboard {
-	height: 100%;
+	margin-top: 2em;
```
Comment (napstr) [inline dashboard/src/components/dashboard/Dashboard.vue:63]:
```suggestion
	height: calc(100% - 2em);
	margin-top: 2em;
```

I like the LogIn component in the middle of the page.

File: dashboard/src/components/dashboard/GuildCard.vue [hunk 2]
```diff
@@ -25,7 +25,7 @@
 				<strong class="is-size-5">{{ guild.name }}</strong>
 			</div>
 		</div>
-		<footer class="card-footer is-size-6">
```
Comment (napstr) [inline dashboard/src/components/dashboard/GuildCard.vue:28]: You can actually have multiple footers on a page.
See https://www.w3schools.com/tags/tag_footer.asp and https://bulma.io/documentation/components/card/

Comment (OtterBoops) [reply dashboard/src/components/dashboard/GuildCard.vue:28]: Point taken. Reverting it now.

**Pull request merged**